.maintain/release-template.md

```diff
@@ -0,0 +1,10 @@
+Darwinia
+===
+
+This is a release template
```

Comment:
The description here needs to be updated.

Reply:
The real content needs to be modified before publishing